sdk/python/foundation-models/system/inference/image-text-to-text/image-text-to-text-online-endpoint.ipynb

```diff
@@ -0,0 +1,440 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Image-text to Text Inference using Online Endpoints\n",
+    "\n",
+    "This sample notebook shows how to deploy `image-text-to-text` type models to an online endpoint for inference.\n",
+    "\n",
+    "### Task\n",
+    "`image-text-to-text` takes as input image-prompt pairs and generates an answer for each pair. The prompt can consist of a single question or a dialog between the user and the model that ends with a question from the user.\n",
+    "\n",
+    "### Model\n",
+    "Models suitable to the `image-text-to-text` task are tagged with `image-text-to-text`. We will use the `llava-7b` model in this notebook. If you opened this notebook from a specific model card, remember to replace the specific model name. If you don't find a model that suits your scenario or domain, you can discover and [import models from HuggingFace hub](../../import/import_model_into_registry.ipynb) and then use them for inference.\n",
+    "\n",
+    "### Inference data\n",
+    "We will use images from the [fridgeObjects](https://cvbp-secondary.z19.web.core.windows.net/datasets/image_classification/fridgeObjects.zip) dataset.\n",
+    "\n",
+    "\n",
+    "### Outline\n",
+    "1. Setup pre-requisites\n",
+    "2. Pick a model to deploy\n",
+    "3. Prepare data for inference\n",
+    "4. Deploy the model to an online endpoint for real time inference\n",
+    "5. Test the endpoint\n",
+    "6. Clean up resources - delete the online endpoint"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1. Setup pre-requisites\n",
+    "* Install dependencies\n",
+    "* Connect to AzureML Workspace. Learn more at [set up SDK authentication](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-setup-authentication?tabs=sdk). Replace  `<WORKSPACE_NAME>`, `<RESOURCE_GROUP>` and `<SUBSCRIPTION_ID>` below.\n",
+    "* Connect to `azureml` system registry"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install llava-torch==1.0.2 --no-deps\n",
```

Comment:
Nit: Can be put in a different notebook cell

Reply:
Done.